frontend/appflowy_flutter/lib/workspace/presentation/home/menu/app/create_button.dart

```diff
@@ -18,9 +18,14 @@ class NewAppButton extends StatelessWidget {
       LocaleKeys.newPageText.tr(),
       fillColor: Colors.transparent,
       hoverColor: Colors.transparent,
-      fontColor: Theme.of(context).colorScheme.onSurfaceVariant,
+      fontColor: Theme.of(context).colorScheme.tertiary,
       onPressed: () async => await _showCreateAppDialog(context),
-      heading: svgWidget("home/new_app", size: const Size(16, 16)),
+      heading: Theme.of(context).brightness == Brightness.light
+          ? svgWidget("home/new_app", size: const Size(16, 16))
+          : svgWidget(
+              "home/new_app_dark",
+              size: const Size(16, 16),
+            ),
```

Comment:
What if the svg just had the transparent cross, instead of white or black? Or does the blue also change?

I am also thinking that using assets like this will make theming harder, eg. different theme color than blue.

Reply:
I've created a SVG we can use:
```
<svg
   width="16"
   height="16"
   viewBox="0 0 16 16"
   fill="none"
   version="1.1"
   id="svg808"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
   <defs
      id="defs812" />
   <path
      id="path930"
      style="fill:#00bcf0;fill-opacity:1"
      transform="rotate(-90)"
      d="M 0,7.9238281 A 8,8 0 0 0 -8.0585938,0 8,8 0 0 0 -16,8.0390625 8,8 0 0 0 -7.9804688,16 8,8 0 0 0 0,8 Z M -4,8 c 0,0.2769988 -0.2230012,0.5 -0.5,0.5 h -3 v 3 c 0,0.277 -0.2230012,0.5 -0.5,0.5 -0.2769988,0 -0.5,-0.223 -0.5,-0.5 v -3 h -3 C -11.777,8.5 -12,8.2769988 -12,8 c 0,-0.2769988 0.223,-0.5 0.5,-0.5 h 3 v -3 C -8.5,4.2230012 -8.2769988,4 -8,4 c 0.2769988,0 0.5,0.2230012 0.5,0.5 v 3 h 3 C -4.2230012,7.5 -4,7.7230012 -4,8 Z" />
</svg>
```
It's paths are compounded and it's done so it's made for use with Flutter SVG, it can even be imported into a fontset as an Icon.

Preview:
<img width="483" alt="Screenshot 2023-03-30 at 20 36 19" src="https://user-images.githubusercontent.com/42929161/228932296-7eb2f247-3b5a-431d-9bbf-e0f54c64a3de.png">

Reply:
Thanks! I appreciate it.